tools/mo/openvino/tools/mo/front/common/partial_infer/utils.py

```diff
@@ -314,3 +314,13 @@ def is_dynamic_slice(s: [slice, int, None]):
     return isinstance(s, slice) and (s.start is dynamic_dimension or
                                      s.stop is dynamic_dimension or
                                      s.step is dynamic_dimension)
+
+
+def reverse_bypass_infer(node, in_ports: List[int] = None):
+    output_shape = node.out_port(0).data.get_shape()
+    if output_shape is not None:
+        if in_ports is None:
+            in_ports = node.in_ports().keys()
```

Comment:
please do require explicit input ports because it is risky to change some input port by mistake.

Reply:
Agreed and specified in_ports explicitly for operation which have several inputs.